public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -62,11 +62,19 @@ define(function(require) {
       this.updateDialog(publishUrl, true);
     }
 
-    bramble.on("fileChange", publisher.showUnpublishedChangesPrompt.bind(publisher));
-    bramble.on("fileDelete", publisher.showUnpublishedChangesPrompt.bind(publisher));
-    bramble.on("fileRename", publisher.showUnpublishedChangesPrompt.bind(publisher));
-    bramble.on("folderRename", publisher.showUnpublishedChangesPrompt.bind(publisher));
-
+    bramble.on("fileChange", function() {
+      publisher.showUnpublishedChangesPrompt();
```

Comment:
> so to preserve that functionality

I think this change is meant to not preserve that functionality as that was what was causing the error :)